### ISSUE TITLE
Unable to upgrade to latest version 6.2.0 via cocoapods

### ISSUE THREAD

**Issue by Mehul18292** [twilio/twilio-voice-ios#70]
### Description

As per given in documentation, i have changed my pod file to below code to get latest TwilioVoice SDK:
```
pod 'TwilioVoice', '~> 6.2.0'`
```

After making call to `pod install` OR `pod update`, it gives this error:
```
CocoaPods could not find compatible versions for pod "TwilioVoice":
  In Podfile:
    TwilioVoice (~> 6.2.0)

Specs satisfying the `TwilioVoice (~> 6.2.0)` dependency were found, but they required a higher minimum deployment target.
```

What other changes i need to do to get latest SDK, Please let us know.

**Comment by bobiechen-twilio** [comment]
Hi @Mehul18292 

Thanks for reaching out. Starting from `6.2.0` we are shipping the Voice iOS SDK in the form of XCFramework for Swift Package Manager compatibility, and have decided to raise the minimum deployment target to `iOS 12` in the podspec. To continue integrating the SDK via Cocoapods, you will have to update the iOS target version in your `Podfile`.

**Comment by piyushtank** [comment]
@Mehul18292 Thanks for reaching out and reporting the problem. We have root caused the problem in the published podspecs. We are working on fixing it, and we will release the SDK with the fix very soon. I apologize for any inconvenience.

**Comment by Mehul18292** [comment]
@bobiechen-twilio My minimum deployment target is iOS 12.0, still it is giving me this error.

**Comment by bobiechen-twilio** [comment]
Hi @Mehul18292 

We have released [Voice iOS 6.2.1](https://www.twilio.com/docs/voice/voip-sdk/ios/changelog#621) and updated the Cocoapods podspec to address the iOS target. Please update your pod repo and pull the latest version.

Thanks again for reaching out and reporting the issue.
-bobie
